lib/client.js

```diff
@@ -356,7 +356,12 @@ Client.prototype.createTopics = function (topics, isAsync, cb) {
     cb = isAsync;
     isAsync = true;
   }
-  validateKafkaTopics(topics);
+  try {
+    validateKafkaTopics(topics);
+  } catch (e) {
+    if (isAsync) return cb(e);
+    throw new e;
```

Comment:
`validateTopicNames` throws a new instance so there's no need to `new` the error here. Looks good otherwise.